app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
```

Comment:
```suggestion

  # POST: /login
  def create
```
Thêm comment routes trên action của controller để tường minh hơn e nhé

Reply:
<img width="696" height="528" alt="image" src="https://github.com/user-attachments/assets/f6fb5e6e-04aa-49e8-8cf6-f8f253aa56d6" />
<img width="524" height="308" alt="image" src="https://github.com/user-attachments/assets/96cb01b8-3c4b-4977-9964-52132915a44c" />
<img width="570" height="574" alt="image" src="https://github.com/user-attachments/assets/07f7c225-202d-426e-b23b-b22b7e20cbd5" />
<img width="525" height="205" alt="image" src="https://github.com/user-attachments/assets/25dc2b50-5229-4aac-9c87-4526e6f40456" />